Database/Scripts/Builders/txt with datapoints/at108a.txt

```diff
@@ -0,0 +1,5 @@
+#Equivalence Ratio, Critical Charge (g tetryl), Critical Energy (J)
+0.78157, 9.94, 44932.0
```

Comment:
These texts files were scraped from the caltech website by Jamal, the only issue is that there are some of these files that needed to be changed manually to work with our scripts (to prevent a considerable amount of if statements for the small amount of outliers). These changes are small such as headers, missing punctuation, improper units, etc

Reply:
As discussed with @TiraFire these changes will be noted in the Confluence article
[**[Report] How To Populate Database using scraped JSONs**](https://databoom.atlassian.net/wiki/spaces/DB/pages/170557454/Report+How+To+Populate+Database+using+scraped+JSONs)